app/components/wallet-warning.tsx

```diff
@@ -6,8 +6,7 @@ export const WalletWarning: FC<FlexProps> = props => {
   return (
     <Flex width="100%" borderRadius="6px" mt="tight" {...props}>
       <Text textStyle="caption" color="ink.600">
-        ⚠️ We're currently resolving a number of issues related to network connectivity and Ledger
-        devices.
+        ⚠️ Some users are experiencing issues relating to network connectivity and Ledger devices.
```

Comment:
Let's frame as "Please check our status page for known issues if you’re experiencing any difficulties" so it's appropriate regardless of the current state of issues.

Reply:
And "Check status" instead of "Read more on our support page" for the link